Assets/Scripts/StandardSamples/UI/Login/UILoginMenu.cs

```diff
@@ -1171,6 +1179,16 @@ private void CreateDeviceCallback(ref Epic.OnlineServices.Connect.CreateDeviceId
             }
         }
 
+        private void ConnectGoogleId()
+        {
+            SignInWithGoogleManager signInWithGoogleManager = new();
+
+            signInWithGoogleManager.GetGoogleIdToken((string token, string username) => 
```

Comment:
Looking at `GetGoogleIdToken`, it seems as though the callback is passed to java. The callback isn't run if there's some critical failure during set up. Should we teach the callback to report in if there's an error, so that can be logged to the user? Or is it enough that inside GetGoogleIdToken exceptions and such are printed?

Reply:
I think having a way to invoke the callback in the error case makes sense. Good catch.